GimmeDatAPI.PlainOldClrObjects/ZaścianekData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace GimmeDatAPI.PlainOldClrObjects
+{
+    public class ZaścianekData
```

Comment:
Nie wiem, czy polskie znaki w nazwach klas/zmiennych to dobry pomysł. Nie jestem wielkim fanem.